src/tests/ov_helpers/ov_snippets_models/include/subgraph_group_normalization.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "snippets_helpers.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class GroupNormalizationFunction : public SnippetsFunctionBase {
+public:
+    explicit GroupNormalizationFunction(const std::vector<PartialShape>& inputShapes, const size_t& numGroup, const float& eps)
+        : SnippetsFunctionBase(inputShapes), num_groups(numGroup), epsilon(eps)  {
+        OPENVINO_ASSERT(input_shapes.size() == 3, "Got invalid number of input shapes");
+    }
+
+protected:
+    std::shared_ptr<ov::Model> initOriginal() const override;
+    std::shared_ptr<ov::Model> initReference() const override;
+    std::shared_ptr<ov::Model> initLowered() const override;
```

Comment:
Please add a schematic representation of the lowered graph, like the one for MHA pattern [here](https://github.com/openvinotoolkit/openvino/blob/718b5a60bf51f051b902e5b274b22cdb438827ee/src/tests/ov_helpers/ov_snippets_models/include/subgraph_mha.hpp#L28-L42)
It would significantly simplify debugging in case the GN decomposition test fails.

Reply:
Yes, it really helpful to understand with an example of decomposed graph, added in [ccaf81b](https://github.com/openvinotoolkit/openvino/pull/23903/commits/ccaf81b6f7db904266b20acf39a39afe9e4caea1). Thank you Ivan!